clusters/build-clusters/02_cluster/assets/admin_cluster-admin-0_list.yaml

```diff
@@ -31,6 +31,10 @@ items:
   - rphillips
   - runcom
   - alexanderConstantinescu
+  - darkmuggle
```

Comment:
Sorry, this is not an option, global cluster admin is restricted to dptp and ppl who need it to help us debug issues with the cluster. Please limit the admin access to a namespace of yours.
/hold

Reply:
@alvaroaleman  I've updated with name-space specific role grant. In retrospect, I totally agree with your nack, there.

Reply:
thank you. It seems that this is not relevant here, but I would like to mention that we generally reserve the right to recreate these clusters if we end up in a situation where fixing them is too hard. Thus I strongly advice to keep the manifests for everything in there you care about versioned in git :)
/lgtm
/approve